pkg/minikube/cruntime/crio.go

```diff
@@ -70,12 +73,18 @@ func (r *CRIO) DefaultCNI() bool {
 
 // Available returns an error if it is not possible to use this runtime on a host
 func (r *CRIO) Available() error {
-	return r.Runner.Run("command -v crio")
+	c := exec.Command("command", "-v", "crio")
```

Comment:
I believe this still needs bash: command is a shell built-in.

Reply:
how about using "which" ?